app/services/auto-approval/checks/has-claimed-less-than-max-times-this-month.js

```diff
@@ -0,0 +1,39 @@
+const claimStatusEnum = require('../../../constants/claim-status-enum')
+const AutoApprovalCheckResult = require('../../domain/auto-approval-check-result')
+
+const CHECK_NAME = 'has-claimed-less-than-max-times-this-month'
+const FAILURE_MESSAGE = 'This claimant has claimed more than the maximum number of times this month'
+
+module.exports = function (autoApprovalData) {
+  if (!autoApprovalData.previousClaims || autoApprovalData.previousClaims.length < 1) {
+    return new AutoApprovalCheckResult(CHECK_NAME, true, '')
+  }
+
+  var date = new Date()
```

Comment:
Instead of using Date maybe use Moment to be consistent, the check on has-claim-less-than-max-times-this-year uses it.